components/chunking/chunk-db-write-impl/src/main/kotlin/net/corda/chunking/db/impl/persistence/CpiPersistence.kt

```diff
@@ -85,5 +85,5 @@ interface CpiPersistence {
      *
      * true: If nothing is in the db we can clearly insert `(any, any)`
      */
-    fun canUpsertCpi(cpiName: String, groupId: String, forceUpload: Boolean = false, cpiVersion: String? = null): Boolean
+    fun canUpsertCpi(cpiName: String, groupId: String, forceUpload: Boolean = false,cpiVersion: String? = null): Boolean
```

Comment:
I'm assuming this change was not intentional.

```suggestion
    fun canUpsertCpi(cpiName: String, groupId: String, forceUpload: Boolean = false, cpiVersion: String? = null): Boolean
```

Reply:
Fixed